src/js/main.js

```diff
@@ -630,24 +633,24 @@
                 mainTune.currentTime = 0;
                 var promise = mainTune.play();
                 if (promise !== undefined) {
-                    promise.then(_ => {
+                    promise.then(() => {
                       // Autoplay started!
-                    }).catch(error => {
+                    }).catch(error => { // jshint ignore:line
                       // Autoplay was prevented.
                     });
                   }
                 break;
             }
             case "end": {
-                if (endTune.played.length == 0) { // To play only once when looping rendering
+                if (endTune.played.length === 0) { // To play only once when looping rendering
                     mainTune.pause();
                     endTune.play();
                 }
                 break;
             }
         }
     }
-    function initMusicPlayer(audioTrack,tune,isLooped) {
```

Comment:
This was meant to have isLooped and have it used in function as:
audioTrack.loop = isLooped;

Reply:
I can fix it later also.

Reply:
Actually, I noticed the same while implementing restarting of a level. So I will take care of this.